docs/version-1.4.0.md

```diff
@@ -0,0 +1,78 @@
+# Version 1.4.0
+
+**Release Name**: CTK 1.4.0
+
+**Upgrade From**: CTK 1.3.0
```

Comment:
we're currently  removing '**Upgrade From**: CTK 1.3.0' Please check with Resham before we remove.